### PR TITLE
fix: Add pull request template to guide users/contributors for when submitting a new pull request

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes:**
* Fixes issue #4  

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

### CODE REVIEW

**Comment by samuko-things** [comment]
> **This pull request makes the following changes:**
> 
> * Fixes issue [Add pull request template to guide users/contributors for when submitting a new pull request #4](https://github.com/code-collabo/node-mongo/issues/4)
> 
> **Checklist:**
> 
> * [x]  File or folder now contains changes as specified in the issue i worked on
> * [x]  I have linked the issue I worked on to this pull request submitted by me
> * [x]  I certify that I ran my checklist.

**Comment by samuko-things** [comment]
@Ifycode, I have just added a pull request template, I would like you to review it

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+**This pull request makes the following changes:**
+* Fixes issue #issueNumberHere
```
Comment (Ifycode) [inline .github/pull_request_template.md:2]: @samuko-things Change this line to:
- Fixes issue code-collabo/repo-name-here#issueNumberHere

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (samuko-things) [reply .github/pull_request_template.md]: i have made the changes

**Comment by samuko-things** [comment]
i have made the changes

**Review by Ifycode — APPROVED** [review]

**Pull request merged**